ComSemApp/templates/ComSemApp/student/assessment_results.html

```diff
@@ -26,46 +35,98 @@ <h4 class="header-title m-t-0 m-b-20">Speaking Practice</h4>
                                     </td>
                                 </tr>
                                 <!--row that shows hidden information-->
-                                <tr id={{attempt.id}} class="hidden_row">  <!--row id corresponds to individual item id-->
-                                    <td colspan=4> <!-- merges all four columns in this row-->
-                                        <p style="margin-left:40px"> <!--indentation for this paragraph-->
-                                            <b>Correct Expression</b><br> 
-                                            {{attempt.correct_formulation}} <br>
-                                            <b>Given Expression</b><br> 
-                                            {{attempt.incorrect_expression}} <br>
-                                            <b>Graded Expression</b> (hover for details about each error)<br> 
-                                            <mark><span title="There could be errors shown here">{{attempt.transcription}}</span></mark><br>
-                                            <b>Audio Recording</b><br>
-                                            <audio controls title="Reformulation" style="width: 200px" class="previous_attempt_audio" src="{{ attempt.audio_path }}"></audio><br>
-                                        </p>
+                                <tr id="{{attempt.id}}" class="hidden_row">
+                                    <td colspan=4 class="row-indent">
+                                        <table style="table-layout: fixed; margin-top: 0px; width: 100%;">
+                                            <thead>
+                                                <tr>
+                                                    <th style="text-align: Left;">Correct Expression</th>
+                                                    <th style="text-align: Left;">Given Expression</th>
+                                                    <th style="text-align: Left;">Graded Expression(hover for details about each error)</th>
+                                                </tr>
+                                            </thead>
+                                            <tbody>
+                                                <tr>
+                                                    <td style="text-align: Left;">
+                                                        {{attempt.correct_formulation}}
+                                                    </td>
+                                                    <td style="text-align: Left;">
+                                                        {{attempt.incorrect_expression}}
+                                                    </td>
+                                                    <td style="text-align: Left;">
+                                                        <mark><span title="There could be errors shown here">{{attempt.transcription}}</span></mark>
```

Comment:
What's with the span title?

Reply:
that's what shows up when you hover over the span